### ISSUE TITLE
Update cloud-integration.md with details on CloudCost instead of CloudUsage

### ISSUE THREAD

**Issue by thomasvn** [kubecost/docs#707]
We recently default disabled CloudUsage in PR https://github.com/kubecost/cost-analyzer-helm-chart/pull/2532.

However our main Cloud Billing Integration doc [`cloud-integration.md`](https://github.com/kubecost/docs/blob/main/cloud-integration.md) currently details how the CloudUsage process works and how to configure it. I think the docs should reflect the fact that CloudCosts is now the default. Some ideas on how we should refactor these docs:

- Replace all mention of CloudUsage in the doc and replace with CloudCost? Have @Sean-Holcomb validate that what's being stated remains true?
- Simply delete the existing section of CloudUsage? Move the "Installation and configuration" section of [`cloud-costs-explorer.md`](https://github.com/kubecost/docs/blob/main/using-kubecost/getting-started/cloud-costs-explorer.md) to this doc?

**Comment by thomasvn** [comment]
Yes, in v1.106 users will no longer be running CloudUsage and will instead be running CloudCost! There won't be visible changes in the UI due to this change though.

Current description of CloudUsage can probably cover most of CloudCost. However under the hood CloudCost may differ and have increased performance enhancements. cc @Sean-Holcomb .

**Comment by srpomeroy** [comment]
The biggest difference is we no longer pull in the entire bill. We are only pulling in a subset of the bill data.

**Comment by bstuder99** [comment]
@srpomeroy Can you elaborate on that, as in, why this change was implemented and what specifically we are pulling in that subset (unless this info varies by each CSP).

**Comment by srpomeroy** [comment]
I don't think we call it out in any of the Cloud Cost specific docs.
https://docs.kubecost.com/using-kubecost/navigating-the-kubecost-ui/cloud-costs-explorer
https://docs.kubecost.com/apis/apis-overview/cloud-cost-api
https://docs.kubecost.com/apis/apis-overview/cloud-cost-api/cloud-cost-metrics

**I need an engineer to keep me honest here (cc: @Sean-Holcomb ).**

First, we will always pull all the information we need to reconcile the price of the resources consumed by a Kubernetes cluster.

The aggregations `Account`, `Provider`, `Invoice Entity`, and `Service` will reflect the total cost for each cloud bill integration.

After those aggregations, when you drill down, the costs you see will be a subset of the entire bill based on the [helm value](https://github.com/kubecost/cost-analyzer-helm-chart/blob/19b26585dca160446c1535518da557dd5f43f6e3/cost-analyzer/values.yaml#L462) `.values.cloudCosts.topNItems`, which is set to 1,000 as a default. This subset is each day's top `n` items by cost.

And then finally, [we have a label list](https://github.com/kubecost/cost-analyzer-helm-chart/blob/19b26585dca160446c1535518da557dd5f43f6e3/cost-analyzer/values.yaml#L458-L461) that can be used to either include or exclude items to be pulled from the bill.

**Comment by bstuder99** [comment]
So when we discuss [Cloud Usage APIs](https://docs.kubecost.com/apis/apis-overview/cloudcost-diagnostic-apis#cloud-usage-apis), have we replaced these with corresponding Cloud Cost APIs?

**Comment by srpomeroy** [comment]
I am unsure how the Cloud Usage APIs are impacted.

**Comment by thomasvn** [comment]
CloudUsage APIs can still be used, but will likely not be needed as much.

CloudCost APIs similarly have `rebuild` and `repair` endpoints ([code ref](https://github.com/kubecost/kubecost-cost-model/blob/v1.106/pkg/cmd/costmodel/costmodel.go#L2099-L2100)), but work is still needed in getting those documented as well.

**Comment by bstuder99** [comment]
> but work is still needed in getting those documented as well.
Is it as simple as updating the names of the endpoints themselves? If this is something that needs to be updated, it would be good to have more information on this.

**Comment by bstuder99** [comment]
For now, I'm slating this content for 1.106 release, and will follow up with a separate issue on the CloudCost APIs
